doc/dev/mgmt/tests.md

```diff
@@ -84,109 +69,96 @@ credentials = UserPassCredentials(
     'user@domain.com',     # Your new user
     'my_smart_password',   # Your password
 )
-```
+``` -->
 
 ### Get a token with Active Directory application and service principal
 
 Follow this detailed tutorial to set up an Active Directory application and service principal:
 https://azure.microsoft.com/documentation/articles/resource-group-create-service-principal-portal/
 
-To use the credentials from Python,
-you need the application ID (a.k.a. client ID),
-authentication key (a.k.a. client secret),
-tenant ID and subscription ID from the Azure portal for use in the next step.
-[This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
-describes where to find them
-(besides the subscription ID,
-which is in the "Overview" section of the "Subscriptions" blade.)
+To use the credentials from Python, you need the application ID (a.k.a. client ID), authentication key (a.k.a. client secret), tenant ID and subscription ID from the Azure portal for use in the next step. [This section of the above tutorial](https://docs.microsoft.com/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key) describes where to find them (besides the subscription ID, which is in the "Overview" section of the "Subscriptions" blade.)
+
+The recommended practice is to store these three values in environment variables called `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET`. To set an environment variable use the following commands:
+```Shell
+$env:AZURE_TENANT_ID=<value>        # PowerShell only
+set AZURE_TENANT_ID=<value>         # Windows CMD (alternatively, use export AZURE_TENANT_ID=<value> to export to the global env)
+export AZURE_TENANT_ID=<value>      # Linux shell only
+```
+*** Note: when setting these variables, do not wrap the value in quotation marks ***
```

Comment:
Is this true?  consider this question for my own education, I just know I've been using vars as so
`$env:SERVICE_BUS_QUEUE_NAME = 'test'`
and it's been working as I'd expect.

Reply:
I just tested this and in powershell it does not need the quotes. In cmd or bash the quotes will be included in the actual value.